app/electron/main.ts

```diff
@@ -282,11 +282,33 @@ function startElecron() {
       width,
       height,
       webPreferences: {
-        nodeIntegration: true,
-        contextIsolation: false,
+        nodeIntegration: false,
+        contextIsolation: true,
+        preload: `${__dirname}/preload.js`
       },
     });
     mainWindow.loadURL(startUrl);
+
+    mainWindow.webContents.on('dom-ready', () => {
```

Comment:
Could this be outside the app, so it can run for the website version too? Or should it only be visible for the app version?

Reply:
I am not sure do we want to show whether there is a release or not on web version?

Reply:
cc @joaquimrocha

Reply:
We don't. At least for now I don't think it's a good UX to just show the updates for the web version.

For the web I imagine we could have a notification mentioning there is a new version available, with a link for refreshing (similar to WhatsApp web does).
However, showing the release notes is a UX that at least I'd like to give control to the operators for whether to show them or not, and until we do that, I prefer not to have it shown.